quantum/template/ps2avrgb/config.h

```diff
@@ -21,17 +21,19 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define VENDOR_ID       0x20A0
 #define PRODUCT_ID      0x422D
+#define DEVICE_VER      0x0001
 #define MANUFACTURER    You
 #define PRODUCT         %KEYBOARD%
+#define DESCRIPTION     A custom keyboard
 
 #define RGBLED_NUM 16
 
 #define MATRIX_ROWS 2
```

Comment:
Agreeing with Drashna here, the smallest known BMC board is a 60%, I would set this to 8x15

```suggestion
#define MATRIX_ROWS 8
```

Reply:
Do you want to put a matching keymap in as well, then?

Reply:
I believe the current template for keymap defaults to a 3x3 or something. We could have it default to a 60 for ps2avrgb boards. 

Please no one make a < 60% BMC board. BMC is dead!!!

Reply:
Point me at the keymap you want and I'll add it.